### PR TITLE
move virheat and snpfreqplot to virolgy 

### PR DESCRIPTION
**Pull request by PlushZ**

### CODE REVIEW

**Comment by bgruening** [comment]
@wm75 you meant Virology should be used isn't it? https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/718

File: tools_iuc.yaml [hunk 1]
```diff
@@ -4966,11 +4962,20 @@ tools:
...
+  
+  - name: snpfreqplot
+    owner: iuc
+    tool_panel_section_label: Virology
 
   - name: varvamp
     owner: iuc
     tool_panel_section_label: Virology
 
+  - name: virheat
```
Comment (wm75) [inline tools_iuc.yaml:4974]: I guess, you need to *move* virheat since your first PR is merged already.

**Comment by wm75** [comment]
@bgruening is moving snpfreqplot (an already installed tool) possible this way?

If yes that's nice, if not I don't care much. Once virheat is tested and fully functional, we will probably hide the old tool anyway.

**Comment by bgruening** [comment]
Yes. But I need to do something manually which is ok. Better to have it consistent.

**Pull request merged**